nano

```diff
@@ -97,6 +97,29 @@ pull() {
 	esac
 }
 
+push() {
+	local targets='packages'
+
+	# Push everything when no target specified.
+	(($#)) || { for target in ${targets//|/ }; do $FUNCNAME $target; done; return; }
+
+	case $1 in
+		package?(s))
+			local dir=packages; local files=packages/*//Config.in; 
+				local config=""; for f in $files; do; 
+				config="$config \n source $f"; done; 
+				[ -d $dir ] && cd $dir &&
+				docker cp . "$NANO_CONTAINER":/root/buildroot/package &&
+				echo Copied packages to $_ directory. &&
+				docker ssh "$NANO_CONTAINER echo "$config" >> /root/buildroot/Config.in &&
```

Comment:
`docker ssh` does not appear to be a [documented command](https://docs.docker.com/engine/reference/commandline/cli/) even in the newest versions.